README_MACOS.md

```diff
@@ -295,6 +295,58 @@ This should be fixed at some point (its a build tool configuration issue). Until
 
 They do however work on Linux and Windows.
 
+Building with native JACK backend
+------------------------------
+
+If you want to use `scsynth` or `supernova` on macOS with JACK, but _without_ the JackRouter driver, you can build them with the native JACK backend. **In that case you will not be able to boot `scsynth`/`supernova` without booting JACK first.** JackRouter allows any macOS application to stream audio to/from JACK, but at the time of writing this (early 2019) it is outdated and the development has stalled.
+
+First you need to install JACK, either through `homebrew`:
+
+```
+brew install jack qjackctl
+```
+
+or by installing the `JackOSX` package. Please note, JACK from `homebrew` is `jack1` and does _not_ include JackRouter. `JackOSX` is `jack2` and it does include JackRouter. `jack1` and `jack2` implement the same API, but you should have only one of them installed at a time.
+
+In order to build with JACK, you need to add the `-DAUDIOAPI=jack` flag to cmake. The full command to configure the build (with supernova) would be:
+
+```
+cmake -G Xcode -DCMAKE_PREFIX_PATH=`brew --prefix qt5` -DSUPERNOVA=ON -DAUDIOAPI=jack ..
```

Comment:
I included this since I personally like full commands I can just copy/paste - I know this is lazy, but I like to have that option. If you insist this is suboptimal, I'm happy to remove that. 

My question is in what case on macOS the command would be different? If you have a custom place for QT you probably know enough to change it, and if not, this is helpful for newbies (which is what I was for quite some time as far as building goes)

Reply:
there are over a dozen options you can pass to cmake to configure your supercollider build, some of them are documented in this very file.

i understand the desire to be beginner friendly but this really isn't that in my opinion, it's just creating clutter. let's teach people how to effectively use the build system if they don't know, rather than giving complete commands for every possible build scenario. if this command were universal or even the common case, you might have a point.

Reply:
> if this command were universal or even the common case, you might have a point.

IMO this was the common case for this particular scenario (building with JACK) but I see your point it might not be so I'll remove it.